boa3/model/builtin/native/contractmanagementclass.py

```diff
@@ -1,40 +1,19 @@
 from __future__ import annotations
 
-from typing import Any, Dict, Optional
+from typing import Any, Dict
 
+from boa3.model.builtin.interop.nativecontract import ContractManagement
+from boa3.model.builtin.native.inativecontractclass import INativeContractClass
 from boa3.model.method import Method
-from boa3.model.property import Property
-from boa3.model.type.classes.classarraytype import ClassArrayType
-from boa3.model.variable import Variable
 
 
-class ContractManagementClass(ClassArrayType):
+class ContractManagementClass(INativeContractClass):
     """
     A class used to represent ContractManagement native contract
     """
 
     def __init__(self):
-        super().__init__('ContractManagement')
-
-        self._variables: Dict[str, Variable] = {}
-        self._class_methods: Dict[str, Method] = {}
-        self._constructor: Method = None
-
-    @property
-    def instance_variables(self) -> Dict[str, Variable]:
-        return self._variables.copy()
-
-    @property
-    def class_variables(self) -> Dict[str, Variable]:
-        return {}
-
-    @property
-    def properties(self) -> Dict[str, Property]:
-        return {}
-
-    @property
-    def static_methods(self) -> Dict[str, Method]:
-        return {}
+        super().__init__('ContractManagement', ContractManagement)
```

Comment:
I saw that `CryptoLibContract` used the getter property instead and it looks more correct because the getter is attributed with the `ContractGetHashMethod`. It doesn't seem like it's necessary tho, since the tests are working.
(This goes for every native `NativeContractClass` except `CryptoLib`)